### PR TITLE
Add new term phrase

### PR DESCRIPTION
**Pull request by vakodrazan**
Hello Team,

I have finished all the requirements for the add screen. You are welcome to review it.

Here is the [github link](https://github.com/onja-org/malagasy-rn-Thallium/tree/add-new-term)
[Add new terms](https://app.clickup.com/t/35bj88)

Thanks in advance

Refs: 35bj88

### CODE REVIEW

File: src/screens/Add.js [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+import {generateId} from '../utils';
+import DropdownArrowIcon from '../icons/select-dropdown-arrow.svg';
+
+export default ({categories, navigation, addNewPhrase, nativeLanguage}) => {
+  const [englishPhrase, setEnglishPhrase] = useState('');
+  const [malagasyPhrase, setMalagasyPhrase] = useState('');
+  const [selectedCategoryValue, setSelectedCategoryValue] = useState('');
+  const dropdownRef = useRef({});
+
+  const isButtonEnable =
```
Comment (starjardin) [inline src/screens/Add.js]: I am not sure about this but I think  button is enable (isButtonEnable = true) if all englishPhrase, malagasyPhrase and selectedCategory are not empty string. It's a small thing but it confuses me. May be you can explain if you think you meant this.
Other than that good job.

File: package.json [hunk 2]
```diff
@@ -13,22 +13,29 @@
...
     "react": "17.0.1",
     "react-native": "0.64.0",
     "react-native-dev-menu": "^4.0.2",
     "react-native-elements": "^3.4.0",
     "react-native-gesture-handler": "^1.10.3",
+    "react-native-get-random-values": "^1.7.0",
+    "react-native-picker-select": "^8.0.4",
     "react-native-safe-area-context": "^3.2.0",
     "react-native-screens": "^3.1.1",
+    "react-native-select-dropdown": "^1.0.6",
```
Comment (andrzej-onja) [inline package.json:28]: please check if you are actually using all the packages

File: src/screens/Add.js (hunk 1 above)
Comment (andrzej-onja) [reply src/screens/Add.js]: isButtonEnable**d**

File: src/screens/Add.js [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+              </View>
+              <Textarea
+                editable={true}
+                phrase={malagasyPhrase}
+                onChange={text => setMalagasyPhrase(text)}
+                placeholder="Enter here"
+              />
+            </View>
+            <Button
+              isDisabled={isButtonEnable}
```
Comment (andrzej-onja) [inline src/screens/Add.js]: this doesnt make sense, rename isButtonEnable to isButtonDisabled or refactor logic to be falsy
and do
`isDisabled={!isButtonEnable}`

**Review by starjardin — APPROVED** [review]

**Review by andrzej-onja — APPROVED** [review]

**Review by Ny-Rianala — APPROVED** [review]

**Pull request merged**